crates/fda/src/main.rs

```diff
@@ -1176,20 +1177,53 @@ async fn pipeline(format: OutputFormat, action: PipelineAction, client: Client)
     }
 }
 
-async fn table_connector(
+async fn connector(
+    format: OutputFormat,
     pipeline_name: String,
-    table_name: &str,
-    connector_name: &str,
+    relation: &str,
+    connector: &str,
     action: ConnectorAction,
     client: Client,
 ) {
+    let rc = client
+        .get_pipeline()
+        .pipeline_name(pipeline_name.clone())
+        .send()
+        .await
+        .map_err(handle_errors_fatal(
+            client.baseurl.clone(),
+            "Failed to get pipeline config",
+            1,
+        ))
+        .map(|response| response.program_info.clone())
+        .unwrap()
+        .unwrap();
+
+    let full_connector_name = format!("{relation}.{connector}");
+    let relation_is_table = rc
+        .input_connectors
+        .iter()
+        .any(|(name, _c)| *name == full_connector_name);
+    let relation_is_view = rc
+        .output_connectors
+        .iter()
+        .any(|(name, _c)| *name == full_connector_name);
+    if !relation_is_table && !relation_is_view {
+        eprintln!("No connector named {connector} found in pipeline {pipeline_name}");
+        std::process::exit(1);
+    }
+
     match action {
         ConnectorAction::Start => {
+            if !relation_is_table {
+                eprintln!("Can't start connector {connector}, not attached to a table");
```

Comment:
Maybe a clearer error message (here and below) would be:
"Cannot start the output connector '{connector}'. Only input connectors (connectors attached to a table) can be started or paused."